flake.nix

```diff
@@ -241,18 +241,38 @@
         rec {
           inherit cargoArtifacts;
           nickel-lang-core = buildPackage { pname = "nickel-lang-core"; };
-          nickel-lang-cli = buildPackage { pname = "nickel-lang-cli"; };
+          nickel-lang-cli = buildPackage {
+            pname = "nickel-lang-cli";
+            extraBuildArgs = "--features format";
+          };
           lsp-nls = buildPackage { pname = "nickel-lang-lsp"; };
 
+          # Static building isn't really possible on MacOS because the system call ABIs aren't stable.
           nickel-static =
             if pkgs.stdenv.hostPlatform.isMacOS
             then nickel-lang-cli
```

Comment:
Should we silently fall back to non static here, or rather `throw` that static binaries aren't supported on MacOS? Wouldn't the former be at risk of being misleading?

Reply:
My main concern was the MacOS CI checking that all derivations in `packages` actually evaluate successfully. I suppose the most principled option would be to remove the `nickel-static` output for `*-darwin` systems.